lib/socket-server.js

```diff
@@ -61,7 +61,7 @@ function onConnection(server, options) {
 
     // only send stats to newly connected clients, if no previous clients have
     // connected and stats has been modified by webpack
-    if (options.stats && options.stats.toJson && !server.clients.length) {
+    if (options.stats && options.stats.toJson && server.clients.size === 1) {
```

Comment:
Is this due to `clients` being a `Set` rather than an `Array`?

Reply:
Yep, the change from `length` to `size` is. 

Now it also expects there to be 1 connected client, instead of 0 like before, since this happens inside "connected" event callback and the new client whose connection caused that event will already be in the Set.

Reply:
OK, that makes sense.